### ISSUE TITLE
Win 1909 "Group By" feature will not stay set at "none" + a couple of minor annoyances 

### ISSUE THREAD

**Issue by CT71** [tablacus/TablacusExplorer#291]
Hey!  Been using since 2017 and I love it, but I've had a couple of minor annoyances that I hope you can help with.  

1 - I just updated to Win 1909 and a new feature is to set you downloads folder to group by date.  It's easy to set it back to group by none, but if you refresh the downloads folder it reverts right back to grouping by date.  I found this on the net incase it helps any:

Downloads now has its own template:

HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Explorer\FolderTypes\{885a186e-a440-4ada-812b-db871b942259}

Its default view groups items by Date Mmodified. 

whereas prior to 1903, it used the Generic template ({5c4f28b5-f869-4e84-8e60-f11db97c5cc7}). Its default view settings have no grouping.

So, the situation is more questiionable design decision than outrigit bug.

 I also think this has confounded many because while Apply to Folders will delete saved views used by regular Explorer windows, it doesn't reset the Common Dialogs if they have already saved a view. But, if the dialogs haven't  already saved a view, they will use the modified template created by Apply to Folders.

So, to cover all bases, it's three steps:

    Set Downloads view to your liking
    Apply to Folders
    Copy & paste the following into a PowerShell window:

$Bags = 'HKCU:\Software\Classes\Local Settings\Software\Microsoft\Windows\Shell\Bags'

$DLID = '{885A186E-A440-4ADA-812B-DB871B942259}'

gci $bags -recurse |

   ?{$_.GetSubkeyNames() -contains $DLID} |

   remove-item -Path { join-path $_.PSPath $DLID }

Press <Enter> to execute the above code.

I tried the above and it did seem to work on File Explorer but not to Tablacus.  Any thoughts?

2 - Another thing after the 1909 update that happened is when I click on a folder and then back out of the folder, it used to be that my cursor would stay on that folder.  Now, when I back out it does not.  This is a pain if you have alot of folders.

3 - This is the last minor annoyance.  I hope I can explain this right.  This one has always been like this from the very start of me using Tablacus.  If I click on a folder and then back out of it, the cursor would stay on that folder but it will always be exactly 3 selections up and out of view of the Tablacus viewing screen.  Meaning, when I back out of the folder, if I take my scroll wheel and scroll down 3 "clicks" the folder that my cursor is on will now come into view on the Tablacus screen.  I really don't know if your gonna understand that or not.  I couldn't figure out how to paste an image to show you what I mean.

Thanks, I hope you can help with these.

**Comment by tablacus** [comment]
1. Groups in Download
I haven't set up anything in particular, but I haven't encountered it.
What folder setting add-ons do you use?
ex. "Remember folder view settings" and/or "Folder settings"

2. back out and cursor.
This is also something I have not encountered.
What methods do you use to back out?
ex. Press [Backspace]key

3. Scroll down 3
I haven't encountered it lately.
If scrolling does not occur, is it okay?
How do you move and how do you get back?
I would like detailed instructions.

https://github.com/tablacus/TablacusExplorer/wiki/report-problem

Regards,

**Comment by CT71** [comment]
1 - Yeah, I've always had "remember folder view settings" from the beginning.  This just started happening after updating to Win 1909.  The text that I copy/pasted was from the Microsoft help forum.  When I applied those settings it worked for File Explorer.  I'm not sure what else I can do, I can't ask the question in the MS forums because it did take care of the problem in File Explorer, but there not going to help with a third party program.  I use refresh in my downloads folder constantly, but now can't.

2 - I back out using the back arrow button, I use my mouse for everything, not the keyboard.  So let's say you have a thousand or more subfolders in a folder.  If I back out, now I have to "search" for the folder again, where as before the cursor would stay on that folder when I backed out.  Even though everything in alphabetized, so I'm not technically "searching" for the folder, it's still a huge pain in the ass to scroll down and find the folder again.  It wastes alot of time to keep doing it that way.  I just realized the only folder this happens with is the "downloads" folder.  So just like problem #1 above, these are both only affecting the downloads folder.

3 - This is a hard one to explain.  Can I post a screenshot?  I don't see anywhere that I can do that.  Let's say that your looking at your own Tablacus on your screen.  And from the top of it to the bottom of it let's say you can see 20 folders on your screen.  If I click open a folder (let's say folder #17 towards the bottom of Tablacus) and then back out of it, the folder that I clicked on will never be showing on your screen.  If you then move your cursor onto the scrollbar (at the very top of the scrollbar where you see the "up arrow") on the right side of Tablacus.........if you click the up arrow three times, folder number 20 will come into view on your screen, then folder 19, then folder 18, then on your third click of the up arrow you will now see folder 17 on your screen.  When you back out of a folder, that folder that you clicked on will always be 3 folders out of your screen.  Everytime.  Exactly 3 folders everytime.  This has always happened this way since the day I started using Tablacus.  I can deal with this problem because it's a small annoyance to deal with because I like Tablacus.  Tablacus allows me to configure it to where I'm only showing 5 buttons on my entire screen (a back and forward button, a home button, a move to, and a new folder).  That's all I have, and that's all I want.  And I can make them very very big.  The other explorer's that I've tried I could never make the buttons as big as I wanted them.  I wear glasses and I like big buttons.  So I'd really like to stay with Tablacus, but these additional problems are getting to be a real pain in the ass to deal with.

**Comment by tablacus** [comment]
For now, please take an add-on "Remember folder view settings" to the top.

![image](https://user-images.githubusercontent.com/5156977/82754581-b281cf00-9e08-11ea-8ac7-06e413c9990c.png)

Would you show me the  options for "Remember folder view settings"?

![image](https://user-images.githubusercontent.com/5156977/82754648-4ce21280-9e09-11ea-8a3b-3d189cd566a8.png)

Regards,

**Comment by CT71** [comment]
I'm sorry, I feel kinda silly, but I cannot figure out how to post a screenshot???  I don't see any icon to post screenshot.

**Comment by tablacus** [comment]
There are two ways to do it.
1. Drag the image file to the text input box.
2. Copy the image (e.g. [Win[] + [Shift] + [S]) and paste it into the text input box.

https://gist.github.com/vinkla/dca76249ba6b73c5dd66a4e986df4c8d
https://help.github.com/en/github/managing-your-work-on-github/file-attachments-on-issues-and-pull-requests

Regards,

**Comment by CT71** [comment]
Ok, so I'm not so stupid.  This is how I was trying to add screenshots last night - neither of these ways will let me post a screenshot.  I use Waterfox and Basilisk and neither will work.

In your first screen I am able to get remember folder view setting to the top like yours, but it doesn't show on the left side under "get addons".  The only way I can get it that way is if I click on "options" in the "remember view settings" then it looks exactly like your second screen except mine is set at 1,0000 instead of 1,000.  I am also using version 1.30 of remember settings.  It took me FOREVER to get Tablacus set up how I wanted it to look, that is why I've never updated anything.  I fear that if I update something I'll lose how I set up Tablacus and I will never be able to get it back how I want it.  Setting up the options was very very very hard for me to figure out.

**Comment by tablacus** [comment]
Why don't you duplicate it and test it on it? 

If you don't have it installed in Program Files, you can copy the entire folder to another location to duplicate the entire environment. 

If you have installed it in Program Files, you can duplicate it by copying it to a non-Program Files location and copying the config folder there as well. 

Regards,

**Comment by CT71** [comment]
I've already gone through everything when I set it up - I simply cannot figure it out.  I copied and pasted the tablacus folder to a different location and whatever changes I made to that location it also changed it in the "original" location.  Basically, in plain english - I'm fucked.  I have no friggin' idea why Win 1909 changed these settings in Tablacus.  I can see why my issue #1 happened, but issue #2 I have no idea.  Worked for the past three years, and now doesn't work.   

What would happen if I downloaded version 20.5.25 and tried different settings in that version?  Will it change anything to the version I'm on right now, which is 17.7.20?  I've spent all day trying other file explorers out - I can't get them to look and work like I want them too.

**Comment by CT71** [comment]
Ok I figured out why I couldn't post an image - a setting in ublock prevented me.

Here's a test folder that has 30 folders in it (not in my "downloads" folder)
![fdfdgfdg](https://user-images.githubusercontent.com/25874701/82849995-3f269e80-9e96-11ea-954a-32c0cb74b7c7.PNG)

If I click on #25 and then back out of it, it will look like this (25 is highlighted in blue even though it doesn't look like it is)

![jkjhkjhk](https://user-images.githubusercontent.com/25874701/82850099-a3496280-9e96-11ea-82ab-99ffaa308749.PNG)

If I do this same test but with that same folder in my "downloads" folder it will look like this when I back out.  It goes back to the beginning of the alphabet.  If you have thousands of folders, which I do, it becomes time consuming and a pain in the ass to always have to get back to that folder again. 

![cvnvncn](https://user-images.githubusercontent.com/25874701/82850164-eacfee80-9e96-11ea-95ea-525f8102a92e.PNG)


Now here's a sample of issue #3 - If I click on folder #4 and back out of it this is what happens - your always brought back to a folder that's always exactly 3 folders down from the folder you clicked on, which is why you see folder #7.  Man, I'm so glad I figured out these damn screenshots, lol.   

 ![retertertrt](https://user-images.githubusercontent.com/25874701/82850349-bdd00b80-9e97-11ea-807e-0902b850327f.PNG)

Issue #1 : If I'm in my d'loads folder and hit refresh this is what it looks like on file explorer - everything is exactly the same:

![fgdfgdfgdf](https://user-images.githubusercontent.com/25874701/82850991-0d173b80-9e9a-11ea-9037-7829705db8e6.PNG)

This is what it looks like in Tablacus after I hit refresh - you have to turn off "group" view everytime.  The information I provided in my first post about the Microsoft forum - that fixed it in file explorer but as you can see it does not fix the problem in Tablacus.  I really hope you can help with these three things.

![hdgfhdh](https://user-images.githubusercontent.com/25874701/82851049-39cb5300-9e9a-11ea-85f8-e858680fff09.PNG)

**Comment by ccfs** [comment]
@CT71,

I suggest you to download the latest version of TE from https://tablacus.github.io/explorer_en.html and install it at a location OTHER THAN "C:\Program Files", to make sure that it uses a separate set of config files. There are numerous fixes and enhancements since version 17.7.20. Your problems might have been fixed in the latest version.

Setting up the latest version of TE for use may be a easier way than updating your existing ancient version of TE.

Regards,

**Comment by tablacus** [comment]
Tablacus Explorer is portable apps.
Copy Tablacus Explorer folder to another location, update it there and try it out.
I'm sure the settings will take over with no problem to new version.
The old version remains in its original location.

Regards,

**Comment by CT71** [comment]
So I brought TE on a thumb drive to work and updated and tried everything out.  Everything works on the computer at work.  Brought it home and I still have those same 3 issues.  My computer at work is not running Win 1909, so there ofcourse would be no "grouping" error.  But it did fix issue #2 at work, but not at home.  And like I said before, issue #3 has always happened since the very first time I used TE.  This is not good, I have no idea what to do next.  These issues do not happen with MS File Explorer so I can't ask there for help.  If you need to see anything let me know since I now know how to take a screenshot.

**Comment by CT71** [comment]
So I just d'loaded the latest version of TE and placed it on my desktop.  I opened it up and I changed absolutely nothing.  I left it exactly like it comes from the beginning and it still did all 3 of those problems.

**Comment by tablacus** [comment]
Finally, I found a way to encounter #3.
I have adjusted #3 on Tablacus Explorer 20.5.28.
Please try to update it.

Regards,

**Comment by CT71** [comment]
Thanks you so much for trying.  Unfortunately it still does not work correctly.  But now, instead of always coming back three folders up, now when you come back the page starts from your very first folder in the alphabet.  If you look above in a post I made a couple of days ago, look at the third screenshot, that what it's doing.  Always goes back to the beginning of the alphabet.  So now, issue #3 is also doing what issue #2 is doing.  Issue 1 and 2 are the ones that make it impossible to work with.  Issue #3 I could live with.  Issue 1 and 2 make it difficult to get work done.  Issue 3 is a minor annoyance but doesn't affect how much work I can get done.  

This is very strange.  I'm not sure how a microsoft update can affect a portable program.  And it only affects it on the downloads page which is even stranger.  Is there no "hidden" folder that Tablacus stores data in??  Or something in the registry??

I was going to say that I cannot use Tablacus anymore and will look for a replacement.........but I just thought of a workaround!!!!  I can move everything out of my downloads folder and set up a new folder as my "home" button folder.  I would have to constantly go back and forth between my downloads folder and my new home folder but that would be a much much better solution than living with issue 1 & 2.  So I'll see how that works out for me and I'll continue to see if any other microsoft update changes it back.  Thanks so much for your help, maybe this new solution will work for me.

**Comment by CT71** [comment]
Ok, so I found an even better solution.  I had never done this before so I was nervous as hell, but I was still in the period where I could rollback from Win 1903 to the previous I was on, 1809.  Issue 1 & 2 were gone and worked like they should!!  I then tried your latest TE and issue 3 WORKS as it should now too!  This is obviously not a permanent solution because I don't know how long I can go on build 1809, but it atleast gives me, and horrible microsoft, some time to sort things out.  Whew!  And whatever the japanese word for whew is, lol!  Big thanks for listening and trying to help me find a solution!!!!  : )

**Comment by tablacus** [comment]
OK, thanks.
Thank you so much for trying it out.

Thank you again,

**Comment by CT71** [comment]
Well I'll probably be back a year or so from now, lol.  Thanks for your patience and continuing to try and help me.  By the way, I'm going to keep the newer TE - I added a couple of new add ons to try out.  The duplicate tab one looked pretty cool.  Thanks man, see ya!

**Comment by kettenfett** [comment]
I would like to point out that #1 is still happening. The grouping will be reset to "modified date" after refreshing the tab (or reopening it). This also means, that folders and files will be dosplayed intermixed, which is just a horrible sight ( shame on you, microsoft ! (╯°□°）╯︵ ┻━┻  )

I'm using TE 2.1.7 with "Remember folder view settings" at the top of the addons list.
![grafik](https://user-images.githubusercontent.com/23098362/148749184-760edc01-5095-4eed-af1e-121eb95e50b6.png)

**Comment by tablacus** [comment]
If you don't want to see all the groups, uncheck Tools -> Options -> Lists -> Styles -> Group.

![image](https://user-images.githubusercontent.com/5156977/148750813-ff9d127d-da44-44d4-85ed-57d7c6953a61.png)

Regards,

**Comment by kettenfett** [comment]
This works, but only fixes part of the windows bug described in OPs #1.

There is still the **sorting** by "modified date" columns.

**Comment by kettenfett** [comment]
UPDATE:
I just found a possible fix for this.

I checked **Tools -> Options -> Lists -> Styles -> Explorer Options**
![grafik](https://user-images.githubusercontent.com/23098362/148753082-b8745709-9ad9-42d9-80b5-4b54edffafe4.png)

(Also uncheck Tools -> Options -> Lists -> Styles -> Group.)

Now, the folder sorting is not reset anymore after refresh (F5).

**Comment by kettenfett** [comment]
So maybe the faulty behaviour was somehow introduced by https://twitter.com/tablacus/status/1478741515648512003 ?
